nodejs/dist/binding.gyp

```diff
@@ -6,7 +6,12 @@
         'src/uWS.cpp',
         'src/addon.cpp'
       ],
-      'cflags_cc': [ '-std=c++11', '-fexceptions' ]
+      'cflags_cc': [ '-std=c++11', '-fexceptions' ],
+      'conditions': [
+        ['OS=="darwin"', {
+          'cflags_cc': [ '-stdlib=libc++', '-mmacosx-version-min=10.7', '-undefined' ]
```

Comment:
'-stdlib=libc++' is the only flag we need here, as we target our own system (I'm also 99% sure node-gyp automatically adds the -undefined dynamic_lookup flag automagically.